src/mbgl/platform/gl.cpp

```diff
@@ -55,6 +59,143 @@ void checkError(const char *cmd, const char *file, int line) {
         throw ::mbgl::gl::Error(err, std::string(cmd) + ": Error GL_" + error + " - " + file + ":" + util::toString(line));
     }
 }
+}
+}
+
+#ifdef GL_TRACK
+#undef glBindTexture
+#undef glDeleteTextures
+#undef glTexImage2D
+#undef glClear
+#undef glShaderSource
+#undef glBufferData
+#undef glBindBuffer
+#undef glDeleteBuffers
+#undef glBufferData
+static unsigned int current_used_bytes = 0;
```

Comment:
Please use camel case variables here and elsewhere in the change (`currentUsedBytes`).